src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+
+};
```

Comment:
because the variable names match the property keys names you want, you can use object shorthand to write this. i.e.
```
const createPerson = (name, age) => {
  return {
    name,
    age
  };
};
```